packages/openapi/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "openapi",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
```

Comment:
This was strange. After I had set up everything else in this PR, I did `yarn docker build backend`. It completed successfully. I ran the container and it threw an error that the require of `./middleware` from `packages/openapi/index.js` could not be found. This seemed wrong; I looked in the `dist` folder in the repo, it was there. I removed the dist folder and did a fresh yarn build. All good. Then I did a `yarn pack`, and sure enough, the package included `dist/index.js` but no other files in the `dist` directory. According to [this conversation](https://github.com/yarnpkg/yarn/issues/2498), specifying a `files` block in the package.json can help yarn pack do the right thing, so I added this and now it works.

Note that the old `pack` would also include things in the root of the `packages/openapi` directory--stuff like the package json and other configs. Those do not appear to me to be needed, but if they are it should be possible to include them here and have them added to the package.